src/NServiceBus.RavenDB/Subscriptions/RavenDbSubscriptionStorage.cs

```diff
@@ -13,26 +12,42 @@ class RavenDbSubscriptionStorage : Feature
 
         protected override void Setup(FeatureConfigurationContext context)
         {
-            context.Container.ConfigureComponent<ISubscriptionStorage>(b =>
+            var disableAggressiveCaching = context.Settings.GetOrDefault<bool>(DoNotAggressivelyCacheSubscriptionsSettingsKey);
+            if (!context.Settings.TryGet(AggressiveCacheDurationSettingsKey, out TimeSpan aggressiveCacheDuration))
             {
-                var store = DocumentStoreManager.GetDocumentStore<StorageType.Subscriptions>(context.Settings, b);
-
-                var persister = new SubscriptionPersister(store);
+                aggressiveCacheDuration = TimeSpan.FromMinutes(1);
+            }
 
-                if (context.Settings.GetOrDefault<bool>(RavenDbSubscriptionSettingsExtensions.DoNotAggressivelyCacheSubscriptionsSettingsKey))
+            if (!disableAggressiveCaching)
```

Comment:
I actually despise bool negation like that but I figure I don't want to change the key and the meaning of it